### DISCUSSION TITLE
no an external tracker

### DISCUSSION THREAD

**Discussion by zhouq123** [PlusToolkit/PlusLib#1192] [Q&A]
Whether to consider implementation the reconstruct method that has not an external tracker?  like the paper <EchoFusion:Tracking and Reconstruction of Objects in 4D Freehand Ultrasound Imaging without External Trackers>

**Comment by ungi** [comment]
I think that would be outside the scope of PLUS. PLUS is designed for data collection and data streaming to other applications that implement complex algorithms like that.
